assets/src/infrastructure/ui/inputs/dateTime/Datepicker/style.scss

```diff
@@ -0,0 +1,5 @@
+.ee-date-picker {
+	[data-baseweb='base-input'] {
```

Comment:
booooo  

...not your fault... i get why you had to do this...

but boooo... like why release something as public but make it difficult to modify?!!?

Reply:
yeah, it's a bit disappointing to see this from a solution backed by such a big company as Uber